src/Components/UI/VerticalNavigation/VerticalNavigation.css

```diff
@@ -0,0 +1,64 @@
+.burger{
+    width: 3rem;
+}
+
+.burger .line{
+    width: 2rem;
+    height: 0.2rem;
+    background-color: var(--color-white);
+    transition: transform 0.3s ease-in-out;
```

Comment:
Suggestion in place of `ease-in-out` you can add
 `transform .3s cubic-bezier(0.68, -0.55, 0.27, 1.55),
-webkit-transform .3s cubic-bezier(0.68, -0.55, 0.27, 1.55);` 
to give a little bouncy effect.

Reply:
Thank youuuuu for the suggestion.